cmake/thisREST.cmake

```diff
@@ -84,7 +84,7 @@ fi
 
 export REST_SOURCE=${CMAKE_CURRENT_SOURCE_DIR}
 export REST_PATH=\\\${thisdir}
-export ROOT_INCLUDE_PATH=\\\${thisdir}/include:${garfieldDir}/include
+export ROOT_INCLUDE_PATH=\\\$ROOT_INCLUDE_PATH:\\\${thisdir}/include
```

Comment:
No!!!

Reply:
You need to define the garfield path in ROOT_INCLUDE_PATH. It will only work in your docker

Reply:
For example, if you add `:${ROOT_INCLUDE_PATH}` behind as you did in next commits, you are hiding a problem in the validation pipeline, because you defined in your docker `ROOT_INCLUDE_PATH` externally, but this will not be the case for most of the users, so my opinion is that is better that we overwrite `ROOT_INCLUDE_PATH`.

Because the only scenario where this would cause problems is that someone else writes another ROOT library that has nothing to do with REST, and wants to use it. Ok, but I believe this will not be the case in the 99.9999%.

Somehow, we are conditioning the validation tests to the way you defined the variables inside the docker, which might not be the same for any user.

I agree with other env variables there is no other way, such as `LD_LIBRARY_PATH`